relay-server/src/services/autoscaling.rs

```diff
@@ -68,7 +68,7 @@ impl Service for AutoscalingMetricService {
                                 }
                             )
                                 .collect();
-                            let worker_pool_utilization = self.async_pool.metrics().total_utilization();
+                            let worker_pool_utilization = self.async_pool.as_ref().map_or(0, |pool| pool.metrics().total_utilization());
```

Comment:
Not sure if it is better here to make the `runtime_utilization` in `AutoscalingData` an option rather than setting 0 here.